### ISSUE TITLE
ng-multiple does not work on md-select

### ISSUE THREAD

**Issue by bmbarker90** [angular/material#9705]
**Actual Behavior**:
 -  `What is the issue? *` When the expression in ng-multiple evaluates to true, md-select-value updates appropriately but md-select-menu does not change to a multi-select.
 -  `What is the expected behavior?` When ng-multiple evaluates to true, it should change md-select-menu to a multi-select.

**CodePen** (or steps to reproduce the issue): *
 -  `CodePen Demo which shows your issue:` http://codepen.io/anon/pen/Zpyzyq?editors=1010
 -  `Details:` ng-multiple set on the Result select evaluates to true when `Multi Select` is selected in the type select. The Result select should be a multi-select, but it is not.

**Angular Versions**: *
 -  `Angular Version:` v1.5.5
 -  `Angular Material Version:` v1.1.1

**Additional Information**:
 -  `Browser Type: *` Chrome
 -  `Browser Version: *` Version 53.0.2785.116 (64-bit)
 -  `OS: *` OS X El Capitan Version 10.11.6
 -  `Stack Traces:`

----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.

**Comment by benreibriongos** [comment]
Your code is working with multiple select with no checkbox but you can select items. I'm using chrome Version 53.0.2785.143 m in windows 10.

**Comment by bmbarker90** [comment]
Right, but checkboxes should be showing. If I just hard-code `multiple="true"` in the html, then checkboxes do show for each option

**Comment by DevVersion** [comment]
Can confirm this. @topherfangio Do you want to take a look at this?

**Comment by zbjornson** [comment]
`ngMultiple` was dropped from angular core in 1.2.0, though, because it didn't work:

https://github.com/angular/angular.js/issues/2113
https://github.com/angular/angular.js/blob/master/CHANGELOG.md#120rc1-spooky-giraffe-2013-08-13
https://github.com/angular/angular.js/commit/d87fa0042375b025b98c40bff05e5f42c00af114
https://github.com/angular/angular.js/issues/1484

(We hit this problem also, but can work-around it.)

**Comment by wingoku** [comment]
Checkbox isn't visible in my md-select either. I have hardcoded the multiple=true

**Comment by Splaktar** [comment]
I am going to use this issue to track the removal of support for `ng-multiple` in AngularJS Material, so that we can match up with AngularJS 1.2.0+. This will be considered a breaking change, so it is targeted to 1.2.0.

Tracking the evaluation of the argument passed into `multiple` and `ng-multiple` will be handled in https://github.com/angular/material/issues/3068.

**Comment by Splaktar** [comment]
Our version of `ng-multiple` has been changed to `md-multiple` in `1.2.0`. We could consider removing it completely in the future.
